.babelrc.js

```diff
@@ -11,5 +11,5 @@ module.exports = {
     ],
     "@babel/preset-flow"
   ],
-  plugins: ["@babel/plugin-proposal-nullish-coalescing-operator"]
+  plugins: ["@babel/plugin-proposal-nullish-coalescing-operator", "@babel/plugin-proposal-optional-chaining"]
```

Comment:
The unit tests are failing because of:
'@babel/plugin-proposal-optional-chaining' from '/home/runner/work/dlq/dlq'

It appears that this plugin is not being installed by the build machine.

Reply:
The plugin package needs to be added to the dev dependencies in package.json

Reply:
Thanks; fixed.